RSDKv5/RSDK/Scene/Legacy/v3/CollisionLegacyv3.cpp

```diff
@@ -2871,15 +2889,16 @@ void RSDK::Legacy::v3::BoxCollision3(int32 left, int32 top, int32 right, int32 b
     sensors[0].YPos     = (collisionBottom << 16) + player->YPos;
     sensors[1].YPos     = sensors[0].YPos;
     sensors[2].YPos     = sensors[0].YPos;
+
     if (player->YVelocity > -1) {
         for (int32 i = 0; i < 3; ++i) {
-            if (((left < sensors[i].XPos) && (sensors[i].XPos < right)) && ((top <= sensors[i].YPos && (player->YPos - player->YVelocity < top)))) {
-                sensors[i].collided = 1;
+            if ((left < sensors[i].XPos && sensors[i].XPos < right) && (top <= sensors[i].YPos && player->YPos - player->YVelocity < top)) {
```

Comment:
I think this one could use parenthesis around all the checks seperately _instead_ of separating X and Y, but ultimately you can decide on this one, it doesn't matter too much.